app/parse.py

```diff
@@ -8,5 +13,26 @@ class Course:
     duration: str
 
 
+def get_course(course: Tag) -> Course:
+    return Course(
+        name=course.select_one("a h3").text,
+        short_description=course.select_one(
+            "p.typography_landingTextMain__Rc8BD.mb-32"
+        ).text,
+        duration=course.select("p")[-2].text
+    )
+
+
+def get_courses_from_page(soup: BeautifulSoup) -> list[Tag]:
+    return soup.select("div.ProfessionCard_cardWrapper__2Q8_V")
+
+
+def get_page(url: str) -> BeautifulSoup():
```

Comment:
The return type annotation for `get_page` should be `BeautifulSoup` instead of `BeautifulSoup()`. The parentheses are not needed here as `BeautifulSoup` is a class, not a function.